### PR TITLE
Add styles to the LeftSide

### PR DESCRIPTION
**Pull request by cristianbgp**
This PR adds styles to all the components from the LeftSide component

### CODE REVIEW

File: client/src/channel-create.js [hunk 1]
```diff
@@ -10,25 +12,153 @@ function ChannelCreate({ submitSendChannel }) {
...
+            <form
+              action=""
+              onSubmit={handleOnSubmit}
+              onReset={handleOnCancel}
+              css={{ display: "flex", flexDirection: "column" }}
+            >
+              <p
+                css={{
+                  fontSize: "2em",
+                  marginBlockStart: "0.67em",
```
Comment (test0n3) [inline client/src/channel-create.js]: Does this label required all this? What is marginBlockStart? I'm not familiar with it.

File: client/src/channel-create.js [hunk 2]
```diff
@@ -10,25 +12,153 @@ function ChannelCreate({ submitSendChannel }) {
...
+                  fontWeight: "bold"
+                }}
+              >
+                Create Channel
+              </p>
+              <p css={{ marginBottom: "20px", color: "#4e4e4e" }}>
+                Channels are where your members communicate. They’re best when
+                organized around a topic — #leads, for example.
+              </p>
+              <label htmlFor="createChannel" css={{ fontWeight: "bold" }}>
```
Comment (test0n3) [inline client/src/channel-create.js]: Wondering if putting the `input` inside the label could help you.

File: client/src/channel-create.js [hunk 3]
```diff
@@ -10,25 +12,153 @@ function ChannelCreate({ submitSendChannel }) {
...
+              <input
+                name="createChannel"
+                type="text"
+                id="createChannel"
+                css={{
+                  lineHeight: "1.5em",
+                  border: "1px solid gray",
+                  fontSize: "1.1em",
+                  fontWeight: "600",
+                  padding: "0.5em",
```
Comment (test0n3) [inline client/src/channel-create.js]: why this with `lineHeight`?

File: client/src/channel-create.js (hunk 1 above)
Comment (cristianbgp) [reply client/src/channel-create.js]: Actually it depends to another property `writing-mode` but in this case it is not necessary. I should use `margin`

File: client/src/channel-create.js (hunk 3 above)
Comment (cristianbgp) [reply client/src/channel-create.js]: It actually grows up a little bit with that property

File: client/src/channel-element.js [hunk 4]
```diff
@@ -1,14 +1,32 @@
...
   }
 
-  return <li onClick={handleClick}># {channelName}</li>;
+  return (
+    <li
+      onClick={handleClick}
+      css={{
+        boxSizing: "border-box",
+        width: "100%",
+        paddingTop: "3px",
```
Comment (test0n3) [inline client/src/channel-element.js]: Just `padding: "3px 0 3px 10px"` for all those padding lines.

File: client/src/channel-create.js (hunk 2 above)
Comment (cristianbgp) [reply client/src/channel-create.js]: It's not necessary, I linked them with the attribute `htmlFor`

File: client/src/channel-element.js (hunk 4 above)
Comment (cristianbgp) [reply client/src/channel-element.js]: Thanks, I will change it

File: client/src/channel-list.js [hunk 5]
```diff
@@ -1,11 +1,20 @@
...
 import ChannelElement from "./channel-element";
 
 function ChannelList({ listChannel }) {
   const channelsData = listChannel;
 
   return (
-    <ul>
+    <ul
+      css={{
+        paddingLeft: "0",
```
Comment (test0n3) [inline client/src/channel-list.js]: Troublesome indeed, browser have some default styles.
`padding: 0,
margin: 0 0 10px 0`

File: client/src/header-name.js [hunk 6]
```diff
@@ -1,11 +1,58 @@
...
-    <>
-      <h1>Codeable</h1>
-      <p>{username}</p>
-    </>
+    <div
+      css={{
+        display: "flex",
+        flexDirection: "column",
+        justifyContent: "space-between",
+        paddingTop: "10px",
```
Comment (test0n3) [inline client/src/header-name.js]: `padding` property is as follows: 
`top right bottom left`
0 (zero) doesn't require units

File: client/src/header-name.js [hunk 7]
```diff
@@ -1,11 +1,58 @@
...
+        paddingLeft: "20px"
+      }}
+    >
+      <p
+        css={{
+          fontSize: "18px",
+          fontWeight: "bold",
+          display: "flex",
+          alignItems: "center",
+          color: "white"
```
Comment (test0n3) [inline client/src/header-name.js]: We have to check which is the proper/best way to declare colors. With strings, hex, rgb, hsl.

File: client/src/channel-create.js (hunk 2 above)
Comment (test0n3) [reply client/src/channel-create.js]: I guess it's fine, both are inline. But label can help you wrapping some elements.

File: client/src/channel-create.js (hunk 3 above)
Comment (test0n3) [reply client/src/channel-create.js]: well, I meant, you could have used just `lineHeight`. Will update `master` to check it.

**Pull request merged**